types/react-sketchapp/index.d.ts

```diff
@@ -1,10 +1,10 @@
-// Type definitions for react-sketchapp 0.13
+// Type definitions for react-sketchapp 0.15
 // Project: https://github.com/airbnb/react-sketchapp
 // Definitions by: Rico Kahler <https://github.com/ricokahler>
 //                 DomiR <https://github.com/DomiR>
 //                 Sascha Zarhuber <https://github.com/saschazar21>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 3.3
```

Comment:
What change caused the minimum compatible version to become 3.3?
When testing locally, it seems like 2.8 still works fine.

Reply:
You're completely right, that was a leftover from local tests with another module.

fixed in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/34637/commits/dcceb8d800c34f28c81d0684d73b55d9afcc6634